src/main/cpp/libclp_ffi_java/common.cpp

```diff
@@ -132,30 +20,25 @@ namespace libclp_ffi_java {
         // Validate the schemas version
         auto schema_version_bytes = get_java_primitive_array_elements<jbyteArray, jbyte>(
                 jni_env, Java_variablesSchemaVersion, JNI_ABORT);
-        if (nullptr == schema_version_bytes) {
-            return;
-        }
         auto schema_version_bytes_length = jni_env->GetArrayLength(Java_variablesSchemaVersion);
         string_view schema_version(size_checked_pointer_cast<char>(schema_version_bytes.get()),
                                    schema_version_bytes_length);
         if (cVariablesSchemaVersion != schema_version) {
-            JavaUnsupportedOperationException::throw_in_java(jni_env,
-                    "Unsupported version for variables schema");
+            throw JavaUnsupportedOperationException(__FILENAME__, __LINE__, jni_env,
+                                                    "Unsupported version for variables schema");
         }
 
         // Validate the encoding methods version
         auto encoding_methods_version_bytes = get_java_primitive_array_elements<jbyteArray, jbyte>(
                 jni_env, Java_variableEncodingMethodsVersion, JNI_ABORT);
-        if (nullptr == encoding_methods_version_bytes) {
-            return;
-        }
         auto encoding_methods_version_length =
                 jni_env->GetArrayLength(Java_variableEncodingMethodsVersion);
         string_view encoding_methods_version(
                 size_checked_pointer_cast<char>(encoding_methods_version_bytes.get()),
                 encoding_methods_version_length);
         if (cVariableEncodingMethodsVersion != encoding_methods_version) {
-            JavaUnsupportedOperationException::throw_in_java(jni_env,
+            throw JavaUnsupportedOperationException(
+                    __FILENAME__, __LINE__, jni_env,
```

Comment:
nit. I will break the string into to two substrings so everything starts after "(".